app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt

```diff
@@ -393,221 +403,226 @@ class ThreeDotMenuMainRobot {
         }
     }
 }
-private fun threeDotMenuRecyclerView() =
-    onView(withId(R.id.mozac_browser_menu_recyclerView))
 
-private fun threeDotMenuRecyclerViewExists() {
-    threeDotMenuRecyclerView().check(matches(isDisplayed()))
-}
+    private fun threeDotMenuRecyclerView() =
```

Comment:
How come we indented all these functions below this? These functions aren't inside of the ThreeDotMenuMainRobot class so, I don't see any particular reason this should be indented. I did go to this file and do reformat code in Android Studio to see if this was a lint issue. I imagine we want to revert this change since the change is unnecessary.

Reply:
it was a lint fix. I can change it back

Reply:
Weird, my reformat didn't indent everything. It did add a new line here tho. Looking at it visually, I still wouldn't expect the indent since these functions are not in a scope/block.